### DISCUSSION TITLE
Issue updating PG User on Kong 3.6.1

### DISCUSSION THREAD

**Discussion by mayankmurari** [Kong/kong#13440] [Help]
I am updating "pg_user" and "pg_ro_user" with Kong version 3.6.1 and getting below error in pre upgrade migration job.

`024/08/01 06:11:39 [warn] 1298#0: *2 [lua] nginx.lua:300: get_ngx_ssl_from_socket_ctx(): note resty.openssl.auxiliary.nginx is using plain FFI and it's only intended to be used in development, consider using lua-resty-openssl.aux-module in production., context: ngx.timerError: attempt to index local 'ssl' (a nil value)`

I tried the same with Kong version 3.7.1 and it works. Is there any known issues with Kong version 3.6.1 which is causing above error?

**Comment by ProBrian** [comment]
Hi @mayankmurari, What's the result you see when you updating `pg_user` on Kong 3.6.1, does the kong service fail to start or there are any functional error? 
The log you attached is a warning and seems irrelevant with user updating, it will be more helpful if you provide more detailed behavior of kong service after you run your migration job.

**Reply by mayankmurari** [reply]
Hi @ProBrian It is the pre-upgrade migration job which is showing above warning and new kong replica is not coming up. However I do not see the same behaviour when I try with Kong version 3.7.1.

**Reply by ProBrian** [reply]
Just to confirm your reproducing steps:

- change the `pg_user` and `pg_ro_user` in `kong.conf` file.
- run `kong migrations bootstrap`
- run `kong start`

Is that right?

**Reply by mayankmurari** [reply]
We are using Kong helm chart for deployment via Argo CD. 
migrations:
  preUpgrade: true
  postUpgrade: true

Following steps are performed.
- change the pg_user and pg_ro_user in kong.conf file
- sync the changes in Kong which triggers pre-upgrade-migrations Jobs. 

wait-for-db log 
`container wait-for-db is not valid for pod kong-pre-upgrade-migrations-88v4qKong`
